pkg/datapath/linux/probes/probes.go

```diff
@@ -196,10 +203,25 @@ func (p *ProbeManager) SystemKernelHz() (int, error) {
 // warnings when optional parameters are not enabled.
 func (p *ProbeManager) SystemConfigProbes() error {
 	config := p.features.SystemConfig
-	// Required
-	if !config.ConfigBpf.Enabled() {
-		return fmt.Errorf("CONFIG_BPF kernel parameter is required")
+
+	// Check Kernel Config is available or not.
+	// We are replicating BPFTools logic here to check if kernel config is available
```

Comment:
Not a licensing expert, but to double-check - we are not changing our license to GPL because of this sentence?

Reply:
I'm not sure I really follow the 'replicating the BPFTools logic' comment here, the linked code is substantially different and not just because of C vs. Go.

The newly added code here is pretty simply figuring out the kernel version then opening a file on the filesystem based on that version. Given the small sample of trivial code that does not have many variations on implementing properly + language difference it's not a direct copy of the original code so I wouldn't expect any licensing implications here.